.github/workflows/biome-check.yaml

```
@@ -139,31 +142,56 @@ jobs:
+
             const originalErrors = '${{ steps.biome-check.outputs.errors }}';
             const originalWarnings = '${{ steps.biome-check.outputs.warnings }}';
-            const remainingIssues = '${{ steps.progress.outputs.remaining_issues }}';
-            const progressPercentage = '${{ steps.progress.outputs.progress_percentage }}';
+            const totalIssues = '${{ steps.biome-check.outputs.total_issues }}';
+
+            const finalIssuesCount = '${{ steps.biome-check.outputs.final_issues_count }}';
+
             const hasIssues = '${{ steps.biome-check.outputs.has_issues }}';
+            const progressPercentage = '${{ steps.progress.outputs.progress_percentage }}';
             
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)